### PR TITLE
[CL-104] List all users

### PR DESCRIPTION
**Pull request by guarde**

### CODE REVIEW

File: __tests__/app.test.ts [hunk 1]
```diff
@@ -145,6 +145,27 @@ describe('🧪 Express Application', () => {
...
 		});
+
+		describe('GET /api/users', () => {
+			it('200: should get all users', () => {
+				return request(app)
+					.get('/api/users')
+					.expect(200)
+					.then(({ body: { success, data, count } }) => {
+						expect(success).toBe(true);
+						expect(count).toBe(2);
```
Comment (stevelw) [inline __tests__/app.test.ts:160]: 3?

**Review by stevelw — CHANGES_REQUESTED** [review]

File: __tests__/app.test.ts (hunk 1 above)
Comment (guarde) [reply __tests__/app.test.ts:160]: 3?

Comment (stevelw) [reply __tests__/app.test.ts:160]: image

Comment (stevelw) [reply __tests__/app.test.ts:160]: image

Comment (guarde) [reply __tests__/app.test.ts:160]: and what does the beforeAll function look like

Comment (stevelw) [reply __tests__/app.test.ts:160]:
```javascript
beforeEach(async () => {
	await seeder();
});
```

Comment (guarde) [reply __tests__/app.test.ts:160]: How weird. It should be something completely different

Comment (stevelw) [reply __tests__/app.test.ts:160]: https://github.com/clowder-cloud/backend/blob/dc9976101f3a1ddba6a74c49261c0c1e0f804da7/__tests__/app.test.ts#L8

Comment (guarde) [reply __tests__/app.test.ts:160]: Look at the same code in the main branch. Seems the PR for that got pulled in but it didnt change that, despite being prior to this branch. This "version control" lark annoys me.

Comment (stevelw) [reply __tests__/app.test.ts:160]: Presumably you made this branch from main before that PR got merged into main

Comment (stevelw) [reply __tests__/app.test.ts:160]: Are you using work trees?

Comment (guarde) [reply __tests__/app.test.ts:160]: Never heard of them before.

Comment (stevelw) [reply __tests__/app.test.ts:160]: Looks like you didn't branch from main when you started?
image

Comment (stevelw) [reply __tests__/app.test.ts:160]: Ignore me, I'm reading that wrong - it doesn't show main like I expected it to, but it looks like you did branch from it properly.

File: src/routes/users.ts [hunk 2]
```diff
@@ -9,5 +9,6 @@ users.get('/:id/cats', controller.getCats);
 users.get('/settings', usernameAuth, controller.getUser);
 users.patch('/settings', usernameAuth, controller.updateUser);
 users.get('/', controller.getAllUsers);
+users.post('/create', controller.createUser);
```
Comment (stevelw) [inline src/routes/users.ts]: Should it be POST to / instead?

**Review by stevelw — APPROVED** [review]

**Pull request merged**